openapi.yaml

```diff
@@ -215,14 +217,16 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ListReportResponse'
+        401:
+          description: Invalid credentials
```

Comment:
Can you change this to `description: Not Authenticated`

Reply:
Sure!